Doc/extending/extending.rst

```diff
@@ -20,6 +20,11 @@ source file by including the header ``"Python.h"``.
 The compilation of an extension module depends on its intended use as well as on
 your system setup; details are given in later chapters.
 
+.. note::
+
+   The documentation is based on multi-phase initialization (:PEP:`489`)
+   since Python 3.14.  See :file:`Modules/xxmodule.c` for more examples.
```

Comment:
I don't think this note meaningfully helps. Multi-phase is supported in all current versions, so we don't need to explicitly mention versions. It should be seen as the default/recommended method for initialisation; single-phase init should be noted as the special case.
```suggestion
```

Reply:
In my recognition, this change is more related to the PEP 734 acception, and 3.13 should be out of scope.  Also, I'm not sure the point of mixing two ways of initialization in one big tutorial.